third-party/realdds/src/dds-device-server.cpp

```diff
@@ -48,30 +48,30 @@ dds_device_server::~dds_device_server()
 
 void dds_device_server::start_streaming( const std::vector< std::pair < std::string, image_header > > & streams_to_open )
 {
-    for ( auto & p : streams_to_open )
-    {
-        auto & stream_name = p.first;
-        auto & header = p.second;
-        auto it = _stream_name_to_server.find( stream_name );
-        if ( it == _stream_name_to_server.end() )
-            DDS_THROW( runtime_error, "stream '" + stream_name + "' does not exist" );
-        auto & stream = it->second;
-        stream->start_streaming( header );
+    for ( auto & p : streams_to_open )
```

Comment:
Just a notice, I see here and on the dds-device-server.h lots of spaces changes.
FYI

Reply:
There were some CRLF changes, so maybe just ignore whitespace?